src/us/kbase/workspace/test/kbase/admin/WorkspaceAdministrationWithHandlersTest.java

```diff
@@ -187,95 +148,6 @@ private void runCommandFail(
 		}
 	}
 	
-	/* ***********************************************
-	 * Non command specific error tests
-	 * ***********************************************
-	 */
-	
-	@Test
-	public void failNotAdmin() throws Exception {
```

Comment:
have most of these tests been converted and moved over into the new test file?

Reply:
All of deleted ones have been, they're almost identical